libs/sdk-ui-dashboard/src/model/store/permissions/permissionsSelectors.ts

```diff
@@ -12,6 +12,32 @@ const selectSelf = createSelector(
  * This selector returns user's permissions in the workspace where the dashboard is stored. It is expected that the
  * selector is called only after the permission state is correctly initialized. Invocations before initialization lead to invariant errors.
  *
+ * See {@link @gooddata/sdk-backend-spi#WorkspacePermission} for all available permissions.
+ *
+ * In case you need multiple permissions available in your application, use common selector.
+ *
+ * If the permission is not supported by GoodData.CN backend, the selector always returns `false` value.
```

Comment:
I would probably flip this line and the previous one. The
```
In case you need multiple permissions available in your application, use common selector.
```
is more relevant to the example than the
```
If the permission is not supported by GoodData.CN backend, the selector always returns `false` value.
```